app.py

```diff
@@ -54,18 +54,31 @@ def config():
         TWILIO_SYNC_SERVICE_SID=os.environ['TWILIO_SYNC_SERVICE_SID'],
     )
 
-@app.route('/token', methods=['POST', 'GET'])
-def token(identity = None):
+@app.route('/token', methods=['GET'])
+def randomToken():
+    return generateToken(fake.user_name())
+
+
+@app.route('/token', methods=['POST'])
+def createToken():
+    # Get the request json or form data
+    content = request.get_json() if request.get_json() else request.form
```

Comment:
I think this could be: `content = request.get_json() or request.form

Reply:
Fixed in a5e8a25
Thanks!